src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -488,10 +488,30 @@ public void DefaultExecuteCmdlet()
                         Zones = this.Zone ?? this.VM.Zones,
                     };
 
+                    Dictionary<string, List<string>> auxAuthHeader = null;
```

Comment:
Need a changelog update to concide with this fix (allow gallery images from other tenants)